src/main/java/org/akhq/modules/KafkaModule.java

```diff
@@ -237,4 +234,8 @@ public SchemaRegistryClient getRegistryClient(String clusterId) {
         return this.connectRestClient.get(clusterId);
     }
 
+    public ProtobufDeserializationTopicsMapping getProtobufDeserializationTopicsMapping(String clusterId) {
```

Comment:
Don't put this logic on `KafkaModule` please, I don't think we need to have deserializer here.
Keep only this logic on `CustomDeserializerRepository`

Reply:
Ok, KafkaModule.getConnection(String cluster) made `public` and `kafkaModule.getConnection(clusterId).getDeserialization()` moved to CustomDwserializerRepository